frontend/src/components/LandingPage/TeamCard/TeamCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  Card, /*CardImg, CardText,*/ CardBody,
+  CardTitle, CardSubtitle, /*Button*/
+} from 'reactstrap';
+import Image from 'react-bootstrap/Image';
+import './TeamCard.css'
+
+const TeamCard = (props) => {
+  console.log(props.headshot)
```

Comment:
Can you remove this line since it was for testing purposes only.

Reply:
done